### DISCUSSION TITLE
sqlitestudio 3.3.1 not running in my 2 machines

### DISCUSSION THREAD

**Discussion by raccon32** [pawelsalawa/sqlitestudio#4023] [Q&A]
I have upgraded from 3.2.1 to 3.3.1 but after downloading, uncompressing the tar.xz file, finally giving permissions to the executable(sqlitestudio) the application never starts.

I tried also to run it using the terminal, but I am getting the following message:
sqlitestudio: command not found
The other file  sqlitestudiocli gives similar message

In addition to that, I ran the following command
file sqlitestudio 

sqlitestudio: ELF 64-bit LSB shared object, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=1f7fedca9156849d6bd405f03f8186e1d95cdf50, stripped

Please advice

Both of my machines are running Ubuntu 20.04 and I have installed 3.2.1 in both (portable)

**Comment by pawelsalawa** [comment]
1. Shouldn't you run it with `./sqlitestudio`, instead of `sqlitestudio`?
2. What does this print? `ldd sqlitestudio`
3. What does this print? `ldd lib/libcore*`

**Comment by raccon32** [comment]
Hi Pawel,What happened is that the program did not run when double clicking in the program or just by selecting "Run Application" from the menu. something changed in that aspect from version 3.2.1 which was the one I first began with to this latest version.
I am running the app by typying ./sqlitestudio
Note also that in version 3.2.1 I was able to create a desktop icon and launcher of the app using this instructions 
Fix “Add to Favorites” for custom apps in Ubuntu

| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Fix “Add to Favorites” for custom apps in Ubuntu

alu

There is no Add to Favorites in Ubuntu dock for custom apps. You will learn how to create a custom launcher and ...
 |

 |

 |


When running the following command in version 3.2.1 this is what I get:/sqlitestudio-3.2.1/SQLiteStudio$ file sqlitestudio
sqlitestudio: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, BuildID[sha1]=e4a16a54a30f257b4b98c2bcddd023548e0f1746, not stripped

compared to the same command in latest version/sqlitestudio-3.3.2/SQLiteStudio$ file sqlitestudio
sqlitestudio: ELF 64-bit LSB shared object, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=6c18eae3f41e3f2c76c7b555b8aac08632894248, stripped
    On Sunday, March 7, 2021, 7:46:07 AM EST, Paweł Salawa <notifications@github.com> wrote:  
 
 

   
   - Shouldn't you run it with ./sqlitestudio, instead of sqlitestudio`?
   - What does this print? ldd sqlitestudio
   - What does this print? ldd lib/libcore*

**Reply by pawelsalawa** [reply]
So...

1. What do you see in the terminal if you run it from terminal? `./sqlitestudio`
2. What does this print? `ldd sqlitestudio`
3. What does this print? `ldd lib/libcore*`

**Comment by raccon32** [comment]
1. running ./sqlitestudio opens the program2. see attached file3. see attached file
    On Monday, March 8, 2021, 9:02:29 AM EST, Paweł Salawa <notifications@github.com> wrote:  
 
 


So...
   
   - What do you see in the terminal if you run it from terminal? ./sqlitestudio
   - What does this print? ldd sqlitestudio
   - What does this print? ldd lib/libcore*

**Comment by raccon32** [comment]
attached files
    On Monday, March 8, 2021, 9:36:02 AM EST, Raul Mosquera <raulysich2@yahoo.com> wrote:  
 
  
1. running ./sqlitestudio opens the program2. see attached file3. see attached file
    On Monday, March 8, 2021, 9:02:29 AM EST, Paweł Salawa <notifications@github.com> wrote:  
 
 


So...
   
   - What do you see in the terminal if you run it from terminal? ./sqlitestudio
   - What does this print? ldd sqlitestudio
   - What does this print? ldd lib/libcore*

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
    
	linux-vdso.so.1 (0x00007ffde5f9f000)
	libcoreSQLiteStudio.so.1 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libcoreSQLiteStudio.so.1 (0x00007f4cf5791000)
	libguiSQLiteStudio.so.1 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libguiSQLiteStudio.so.1 (0x00007f4cf50c2000)
	libQt5Widgets.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libQt5Widgets.so.5 (0x00007f4cf4860000)
	libQt5Network.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libQt5Network.so.5 (0x00007f4cf44b2000)
	libQt5Core.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libQt5Core.so.5 (0x00007f4cf3cbc000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f4cf3abf000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f4cf3aa4000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f4cf38b2000)
	libQt5Script.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libQt5Script.so.5 (0x00007f4cf342b000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f4cf3408000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f4cf32b9000)
	libQt5Gui.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libQt5Gui.so.5 (0x00007f4cf2986000)
	libGL.so.1 => /usr/lib/x86_64-linux-gnu/libGL.so.1 (0x00007f4cf28fe000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f4cf28e2000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f4cf28dc000)
	libgssapi_krb5.so.2 => /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f4cf288f000)
	libicui18n.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libicui18n.so.56 (0x00007f4cf23f4000)
	libicuuc.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libicuuc.so.56 (0x00007f4cf203c000)
	libicudata.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/./lib/libicudata.so.56 (0x00007f4cf0659000)
	libgthread-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgthread-2.0.so.0 (0x00007f4cf0654000)
	libglib-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f4cf052b000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f4cf5fc7000)
	libGLdispatch.so.0 => /usr/lib/x86_64-linux-gnu/libGLdispatch.so.0 (0x00007f4cf0471000)
	libGLX.so.0 => /usr/lib/x86_64-linux-gnu/libGLX.so.0 (0x00007f4cf043d000)
	libkrb5.so.3 => /usr/lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f4cf0360000)
	libk5crypto.so.3 => /usr/lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f4cf032f000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f4cf0328000)
	libkrb5support.so.0 => /usr/lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f4cf0319000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f4cf02a4000)
	libX11.so.6 => /usr/lib/x86_64-linux-gnu/libX11.so.6 (0x00007f4cf0167000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f4cf0160000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f4cf0144000)
	libxcb.so.1 => /usr/lib/x86_64-linux-gnu/libxcb.so.1 (0x00007f4cf011a000)
	libXau.so.6 => /usr/lib/x86_64-linux-gnu/libXau.so.6 (0x00007f4cf0112000)
	libXdmcp.so.6 => /usr/lib/x86_64-linux-gnu/libXdmcp.so.6 (0x00007f4cf010a000)
	libbsd.so.0 => /usr/lib/x86_64-linux-gnu/libbsd.so.0 (0x00007f4cf00f0000)

lib/libcoreSQLiteStudio.so:
	linux-vdso.so.1 (0x00007ffc7f1fc000)
	libQt5Script.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Script.so.5 (0x00007f9e2f839000)
	libQt5Network.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Network.so.5 (0x00007f9e2f48b000)
	libQt5Core.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Core.so.5 (0x00007f9e2ec95000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f9e2ec58000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f9e2ea77000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f9e2e926000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f9e2e90b000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f9e2e719000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f9e2e6fd000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f9e2e6f7000)
	libgssapi_krb5.so.2 => /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f9e2e6aa000)
	libicui18n.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicui18n.so.56 (0x00007f9e2e20f000)
	libicuuc.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicuuc.so.56 (0x00007f9e2de57000)
	libicudata.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicudata.so.56 (0x00007f9e2c474000)
	libgthread-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgthread-2.0.so.0 (0x00007f9e2c46f000)
	libglib-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f9e2c346000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f9e302e3000)
	libkrb5.so.3 => /usr/lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f9e2c267000)
	libk5crypto.so.3 => /usr/lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f9e2c236000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f9e2c22f000)
	libkrb5support.so.0 => /usr/lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f9e2c220000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f9e2c1ad000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f9e2c1a4000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f9e2c188000)
lib/libcoreSQLiteStudio.so.1:
	linux-vdso.so.1 (0x00007ffe529fc000)
	libQt5Script.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Script.so.5 (0x00007fa892403000)
	libQt5Network.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Network.so.5 (0x00007fa892055000)
	libQt5Core.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Core.so.5 (0x00007fa89185f000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fa891822000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fa891641000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fa8914f0000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fa8914d5000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fa8912e3000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007fa8912c7000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fa8912c1000)
	libgssapi_krb5.so.2 => /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007fa891274000)
	libicui18n.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicui18n.so.56 (0x00007fa890dd9000)
	libicuuc.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicuuc.so.56 (0x00007fa890a21000)
	libicudata.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicudata.so.56 (0x00007fa88f03e000)
	libgthread-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgthread-2.0.so.0 (0x00007fa88f039000)
	libglib-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007fa88ef10000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fa892ead000)
	libkrb5.so.3 => /usr/lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007fa88ee31000)
	libk5crypto.so.3 => /usr/lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007fa88ee00000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007fa88edf9000)
	libkrb5support.so.0 => /usr/lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007fa88edea000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007fa88ed77000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007fa88ed6e000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007fa88ed52000)
lib/libcoreSQLiteStudio.so.1.0:
	linux-vdso.so.1 (0x00007ffee01de000)
	libQt5Script.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Script.so.5 (0x00007ffa3ad16000)
	libQt5Network.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Network.so.5 (0x00007ffa3a968000)
	libQt5Core.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Core.so.5 (0x00007ffa3a172000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007ffa3a135000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007ffa39f54000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007ffa39e03000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007ffa39de8000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007ffa39bf6000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007ffa39bda000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007ffa39bd4000)
	libgssapi_krb5.so.2 => /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007ffa39b87000)
	libicui18n.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicui18n.so.56 (0x00007ffa396ec000)
	libicuuc.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicuuc.so.56 (0x00007ffa39334000)
	libicudata.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicudata.so.56 (0x00007ffa37951000)
	libgthread-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgthread-2.0.so.0 (0x00007ffa3794c000)
	libglib-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007ffa37823000)
	/lib64/ld-linux-x86-64.so.2 (0x00007ffa3b7c0000)
	libkrb5.so.3 => /usr/lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007ffa37744000)
	libk5crypto.so.3 => /usr/lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007ffa37713000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007ffa3770c000)
	libkrb5support.so.0 => /usr/lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007ffa376fd000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007ffa3768a000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007ffa37681000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007ffa37665000)
lib/libcoreSQLiteStudio.so.1.0.0:
	linux-vdso.so.1 (0x00007ffe809fe000)
	libQt5Script.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Script.so.5 (0x00007f3ee9119000)
	libQt5Network.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Network.so.5 (0x00007f3ee8d6b000)
	libQt5Core.so.5 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libQt5Core.so.5 (0x00007f3ee8575000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f3ee8538000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f3ee8357000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f3ee8206000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f3ee81eb000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f3ee7ff9000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f3ee7fdd000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f3ee7fd7000)
	libgssapi_krb5.so.2 => /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2 (0x00007f3ee7f8a000)
	libicui18n.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicui18n.so.56 (0x00007f3ee7aef000)
	libicuuc.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicuuc.so.56 (0x00007f3ee7737000)
	libicudata.so.56 => /home/lab-01-administrator/Lab-01-IDBit-Administration/Lab-04-software-installation/sqlitestudio-3.3.2/SQLiteStudio/lib/libicudata.so.56 (0x00007f3ee5d54000)
	libgthread-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libgthread-2.0.so.0 (0x00007f3ee5d4f000)
	libglib-2.0.so.0 => /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0 (0x00007f3ee5c26000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f3ee9bc3000)
	libkrb5.so.3 => /usr/lib/x86_64-linux-gnu/libkrb5.so.3 (0x00007f3ee5b47000)
	libk5crypto.so.3 => /usr/lib/x86_64-linux-gnu/libk5crypto.so.3 (0x00007f3ee5b16000)
	libcom_err.so.2 => /lib/x86_64-linux-gnu/libcom_err.so.2 (0x00007f3ee5b0f000)
	libkrb5support.so.0 => /usr/lib/x86_64-linux-gnu/libkrb5support.so.0 (0x00007f3ee5b00000)
	libpcre.so.3 => /lib/x86_64-linux-gnu/libpcre.so.3 (0x00007f3ee5a8d000)
	libkeyutils.so.1 => /lib/x86_64-linux-gnu/libkeyutils.so.1 (0x00007f3ee5a84000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f3ee5a68000)

**Reply by pawelsalawa** [reply]
Did I understand that `./sqlitestudio` opnes the program? If yes, then is there still an issue?

**Comment by raccon32** [comment]
Pawel,
the "issue" is that in order to run the program you need to open  the terminal and type. in version 3.2.1 I just double-clicked the name and then it opened up (no terminal involved).

    On Monday, March 8, 2021, 9:55:00 AM EST, Paweł Salawa <notifications@github.com> wrote:  
 
 


Did I understand that ./sqlitestudio opnes the program? If yes, then is there still an issue?
